reference/6/Microsoft.PowerShell.Management/Set-Service.md

```diff
@@ -88,16 +93,19 @@ Description : Transfers files in the background using idle network bandwidth.
 
 These commands change the description of the BITS service and then display the result.
 
-These commands use the `Get-CimInstance` cmdlet to get the **Win32_Service** object for the service, because the **ServiceController** object that `Get-Service` returns does not include the service description.
+These commands use the `Get-CimInstance` cmdlet to get the **Win32_Service** object for the
+service, because the **ServiceController** object that `Get-Service` returns does not include the
+service description.
 
 ### Example 4: Start a service
 
 ```powershell
 Set-Service -Name "winrm" -Status Running -PassThru
 ```
 
-This command starts the WinRM service.
-The command uses the Status parameter to specify the desired status, which is running, and the *PassThru* parameter to direct `Set-Service` to return an object that represents the WinRM service.
+This command starts the WinRM service. The command uses the Status parameter to specify the desired
+status, which is running, and the *PassThru* parameter to direct `Set-Service` to return an object
```

Comment:
PassThru should be bold, and not italic, per style guide.  Status should also be bolded for consistency.

Reply:
fixed